src/db.js

```diff
@@ -416,43 +417,67 @@ var Database = {
   deleteJSONWebTokensForUser: function(userId) {
     assert(typeof userId === 'number');
     return this.run(
-      'DELETE FROM jsonwebtoken_to_user WHERE user = ?',
+      'DELETE FROM jsonwebtokens WHERE user = ?',
       [userId]
     );
   },
 
   /**
-   * Insert a JSONWebToken into the database and return it's primary key.
+   * Insert a JSONWebToken into the database
+   * @param {JSONWebToken} token
+   * @return {Promise<number>} resolved to JWT's primary key
    */
   createJSONWebToken: async function(token) {
-    const {keyId, user, publicKey, issuedAt} = token;
+    const {keyId, user, publicKey, issuedAt, payload} = token;
     const result = await this.run(
-      'INSERT INTO jsonwebtoken_to_user (keyId, user, issuedAt, publicKey) ' +
-      'VALUES (?, ?, ?, ?)',
-      [keyId, user, issuedAt, publicKey]
+      'INSERT INTO jsonwebtokens (keyId, user, issuedAt, publicKey, payload) ' +
+      'VALUES (?, ?, ?, ?, ?)',
+      [keyId, user, issuedAt, publicKey, JSON.stringify(payload)]
     );
     assert(typeof result.lastID === 'number');
     return result.lastID;
   },
```

Comment:
Presumably we need a mechanism to actually expire the old temporary tokens. I noticed that I have a bunch in my database now. Perhaps an issue should be created?

Reply:
Yes, I left out a "DROP TABLE IF EXISTS jsonwebtoken_to_user" because it would make flipping between this and master difficult, but it should be added in the future. I filed #448 to keep track